models/examples/model_governance/example_private_finance_model_v2.sql

```diff
@@ -0,0 +1,3 @@
+select 
+    * exclude priority_code
```

Comment:
WHAT??? I didn't know you could do a select * and exclude a column. super cool!

Reply:
yeah it's a snowflake thing! Idk when they released it but it's the first time I used it :)